packages/editor/src/plugin/helpers/get-plugin-title.tsx

```diff
@@ -3,8 +3,12 @@ import { useEditorStrings } from '@serlo/frontend/src/contexts/logged-in-data-co
 export function getPluginTitle(
   pluginStrings: ReturnType<typeof useEditorStrings>['plugins'],
   pluginType: string
-) {
+): string {
   return Object.hasOwn(pluginStrings, pluginType)
-    ? pluginStrings[pluginType as keyof typeof pluginStrings].title
+    ? (
+        pluginStrings[pluginType as keyof typeof pluginStrings] as {
+          title: string
+        }
+      ).title
```

Comment:
This change was necessary to fix eslint errors in the file

Reply:
The eslint error happened because of this change: https://github.com/serlo/frontend/pull/3986/files?short_path=c59cfd3#diff-e08687cc24fc586c93a42e9fa2ebb10f4a4f3e07819cf1e54b3b9cb8f3699b09R680-R681

Could you consider another place for those two strings? Like, instead of editor.plugins, create a new namespace editor.addPluginsModal? And then the a11y modal title string could go there as well?

Reply:
Fixed, thanks